_includes/custom-head.html

```diff
@@ -7,18 +7,18 @@
 
 
 {%- include favicons.html -%}
-<link href="https://unpkg.com/@primer/css/dist/primer.css" rel="stylesheet" />
-<link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
+<link href="https://cdn.jsdelivr.net/npm/@primer/css/dist/primer.css" rel="stylesheet" />
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free/css/all.css">
```

Comment:
Can someone comment or give me some background on why fontawesome is hosted under `fortawesome` with an `r` ?  It seems a bit odd, but then again I don't have any expertise in this area

Reply:
Both sites are often inaccessible in my area.

Reply:
There are several prices for the font fortawesome 5.

Reply:
Again, what about we stick to the way listed in [the official docs](https://fontawesome.com/account/cdn):

```html
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css" integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous">
```

![image](https://user-images.githubusercontent.com/1483922/93911898-dfab9480-fcb7-11ea-8649-456cecd9c85a.png)

Reply:
I can't open the use.fontawesome.com，which slows down access to fastpages.

Reply:
ok thanks for your patience I think fortawesome is fine.  Thanks for educating me on this

Would love to learn why some things are not accessible for some people!  How can  I test this better in the future?